google/auth/iam.py

```diff
@@ -35,11 +35,14 @@
     http_client.GATEWAY_TIMEOUT,
 }
 
+_DEFAULT_UNIVERSE_DOMAIN = ["googleapis.com"]
```

Comment:
There is already a constant in base class: https://github.com/googleapis/google-auth-library-python/blob/main/google/auth/credentials.py#L28.

Use it in init like https://github.com/googleapis/google-auth-library-python/blob/484c8db151690a4ae7b6b0ae38db0a8ede88df69/google/auth/external_account_authorized_user.py#L78

Reply:
Yep that was not needed